packages/react-native-editor/__device-tests__/gutenberg-editor-block-insertion-2.test.js

```diff
@@ -27,21 +27,9 @@ describe( 'Gutenberg Editor tests for Block insertion 2', () => {
 		);
 
 		await headingBlockElement.click();
-
 		await editorPage.addNewBlock( blockNames.separator );
 
-		const expectedHtml = `<!-- wp:heading -->
-<h2></h2>
-<!-- /wp:heading -->
-
-<!-- wp:separator -->
-<hr class="wp-block-separator has-alpha-channel-opacity"/>
-<!-- /wp:separator -->
-
-<!-- wp:list -->
-<ul><li></li></ul>
-<!-- /wp:list -->`;
-
+		const expectedHtml = `${ headingExpectedHTML }\n\n${ separatorExpectedHTML }\n\n${ listExpectedHTML }`;
```

Comment:
Nitpick: We could use get the expected HTML using an array and the `join` function.
```suggestion
		const expectedHtml = [
			headingExpectedHTML,
			separatorExpectedHTML,
			listExpectedHTML,
		].join( '\n\n' );
```